packages/deployments/contracts/contracts/core/connext/helpers/StableSwap.sol

```diff
@@ -83,7 +83,8 @@ contract StableSwap is IStableSwap, OwnerPausableUpgradeable, ReentrancyGuardUpg
 
     uint256[] memory precisionMultipliers = new uint256[](decimals.length);
 
-    for (uint256 i = 0; i < _pooledTokens.length; i++) {
+    uint256 len = _pooledTokens.length;
+    for (uint256 i = 0; i < len; i++) {
```

Comment:
idk if we're conforming with this across this whole file, but we should be using the ideal for loop conditional pattern:
```
for (uint256 i; i < len; ) {
  ...
  unchecked {
    ++i;
  }
}
```

Reply:
This comment applies to multiple `for` loops in the section, but isn't really the responsibility of this specific PR. So I'm going to label this as a nit (feel free to fix all instances) and approve

Reply:
Fixed this. Changed quite a few loops so this needs another look.